recipes/quirc/all/patches/fix_allocation_table_null_size.patch

```diff
@@ -0,0 +1,22 @@
+--- a/lib/identify.c
++++ b/lib/identify.c
+@@ -196,9 +196,9 @@ static void threshold(struct quirc *q)
+ 		threshold_s = THRESHOLD_S_MIN;
+ 
+ 	for (y = 0; y < q->h; y++) {
+-		int row_average[q->w];
++		int *row_average = malloc(q->w * sizeof(int));
+ 
+-		memset(row_average, 0, sizeof(row_average));
```

Comment:
I think patch is incorrect as it doesn't check the result of the **malloc** call, so it may crash. pls fix.

Reply:
fixed with modifications introduced in 2 commits on quirc repo.